### DISCUSSION TITLE
关于页眉和算法包的问题

### DISCUSSION THREAD

**Discussion by swpu-kewei** [tuna/thuthesis#718] [Q&A] [answered]
1.如何控制摘要、目录页不显示页眉，仅显示页脚的页编码？
尝试了很多{plain}都没有效果。
2.{algorithmic}+{algorithm}环境下，可否显示算法的缩进线（比如for-end for之间的垂直缩进线），尝试了很多都不行，好像要另一个2e的算法包，但是调用后编译报错

3.{algorithmic}+{algorithm}环境下，\COMMENT增加算法注释，但是注释内容显示时被{}括起来，可否换一个格式，比如/* XXXX */这样的，因为大括号容易产生误解。如何更改

**Comment by zepinglee** [comment, marked as answer]
1. 学校要求了固定的格式，不要乱改。
2. `algorithms` 好像无法实现缩进线的效果。建议给出 `algorithm2e` 的示例和报错信息。
3. `algorithms` 的文档中没有提到，但是可以在 `algorithmic.sty` 的源码中找到 `\newcommand{\algorithmiccomment}[1]{\{#1\}}`，重定义这个命令就行了。
